cranelift/codegen/src/isa/x86/enc_tables.rs

```diff
@@ -1856,12 +1856,13 @@ fn convert_i64x2_imul(
     }
 }
 
+/// Expand a `tls_value` instruction.
 fn expand_tls_value(
     inst: ir::Inst,
     func: &mut ir::Function,
     _cfg: &mut ControlFlowGraph,
     isa: &dyn TargetIsa,
-) {
+) -> bool {
```

Comment:
Why this change?

Reply:
Ah, this (and `legalizer/tls.rs`) was leftover from an earlier version of the change; reverted to original. Thanks!